### PR TITLE
add depth loss

### PR DESCRIPTION
**Pull request by BosonicJustin**

### CODE REVIEW

File: losses.py [hunk 1]
```diff
@@ -44,17 +45,15 @@ def __init__(self, lambda_opacity=1e-3, lambda_distortion=1e-3):
...
-    def forward(self, results, target, **kwargs):
-        d = {}
-        d['rgb'] = (results['rgb']-target['rgb'])**2
+    def forward(self, results, target):
+        o = results['opacity'] + 1e-10
 
-        o = results['opacity']+1e-10
-        # encourage opacity to be either 0 or 1 to avoid floater
-        d['opacity'] = self.lambda_opacity*(-o*torch.log(o))
+        l = dict(
```
Comment (mschneider456) [inline losses.py:53]: What does l stand for?

Comment (BosonicJustin) [reply losses.py:53]: loss
